src/rust-crypto/verification.ts

```diff
@@ -658,6 +673,14 @@ export class RustSASVerifier extends BaseRustVerifer<RustSdkCryptoJs.Sas> implem
 
     /** if we can now show the callbacks, do so */
     protected onChange(): void {
+        if (this.started) {
```

Comment:
Yes, if we already accepted with that Rust SAS struct, then `this.inner.accept()` will return `undefined`, so it will be a no-op.  I'll clarify in the comment.

Reply:
can we not intercept `setInner` and just do this logic then?

Reply:
added an `onSetInner` function that gets called when `setInner` actually changes the `inner`.